app/src/main/java/com/sdp13epfl2021/projmag/activities/ProfilePageActivity.kt

```diff
@@ -78,7 +71,7 @@ class ProfilePageActivity : AppCompatActivity() {
             findViewById<EditText>(R.id.profile_phone_number).setText(profile.phoneNumber)
             findViewById<EditText>(R.id.profile_sciper).setText(profile.sciper.toString())
         } else {
-            Log.d(ContentValues.TAG, "NULL")
+            Toast.makeText(this, "Failed to load profile", Toast.LENGTH_LONG).show()
```

Comment:
also a string resource
And as it is the same toast message you could save it and just call `show()` on it to avoid duplication

Reply:
I agree